sample/ws-chat-server.c

```diff
@@ -233,6 +233,9 @@ main(int argc, char **argv)
 	base = event_base_new();
 
 	sig_int = evsignal_new(base, SIGINT, signal_cb, base);
+	if (sig_int == NULL) {
```

Comment:
Should clean the base in this case, i.e. `goto cleanup`

```
cleanup:
    if (base)
        event_base_free(base);
```

And should check the base then as well

Reply:
Got it, I'll fix it